xapian-bindings/configure.ac

```diff
@@ -1108,6 +1141,50 @@ if test no != "$with_lua" ; then
   fi
 fi
 
+if test no != "$with_go" ; then
+  AC_PATH_PROG([GO],["go"],[])
+  if test -n "$GO" ; then
+    AC_MSG_CHECKING(for goroot)
+    GOROOT=`"${GO}" env GOROOT`
+    GOPATH=`"${GO}" env GOPATH`
+    AC_SUBST(GOROOT)
+    AC_SUBST(GOPATH)
+    GO_CORE="without-core"
+    if $XAPIAN_CONFIG --libs >/dev/null 2>/dev/null ; then
+      GO_CORE="with-core"
+      GO_CXX=`$XAPIAN_CONFIG --cxxflags`
+      GO_LIB=`$XAPIAN_CONFIG --libs`
+      AC_SUBST([GO_CXX])
+      AC_SUBST([GO_LIB])
+    fi
+    AC_SUBST([GO_CORE])
+    if test -n "${GOROOT}" ; then
+      AC_MSG_RESULT([${GOROOT}])
+      AC_MSG_CHECKING(for gopath)
+      if test -n "${GOPATH}" ; then
+        AC_MSG_RESULT([${GOPATH}])
+      else
+        AC_MSG_RESULT(no)
+      fi
+    else
+      AC_MSG_RESULT(no)
+    fi
+    if test -z "${GOROOT}" ; then
+      AC_MSG_ERROR([goroot not set correctly])
+    fi
+  fi
+  USE_CLANG_LIB=`echo ${CXX} | grep stdlib`
+  AC_MSG_RESULT([${USE_CLANG_LIB}])
+  if test -n "${USE_CLANG_LIB}" ; then
+    AC_MSG_RESULT([${USE_CLANG_LIB}])
+    AC_SUBST(USE_CLANG_GO_LIB,'-lc++')
```

Comment:
Copying over a comment from the old PR which was never responded to:

> This also looks very dubious to me, and could misdetect if the stdlib is present elsewhere - e.g. GXX=/home/stdlib/bin/gcc, or misdetect the other way if CXXFLAGS=-stdlib=libc++ is used instead of CXX='clang -stdlib=libc++'.
>
> What is this actually trying to achieve?

I realise you may well not know, but if so it seems better to drop this and see what (if anything) breaks so we actually understand the need for it, and can perhaps address it better.